website/addons/owncloud/serializer.py

```diff
@@ -0,0 +1,65 @@
+from website.addons.base.serializer import StorageAddonSerializer
+from website.addons.owncloud.settings import DEFAULT_HOSTS
+from website.util import web_url_for
+from website.addons.owncloud.utils import ExternalAccountConverter
```

Comment:
@kwierman: I think this was accidentally left in.  This module doesn't exist in the PR and keeps the OSF from starting up.  Same issue in `views.py`.